### PR TITLE
HBASE-22329 Fix for warning The parameter forkMode is deprecated since

### PR DESCRIPTION
**Pull request by dbist**

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/PreCommit-HBASE-CONNECTORS-Build/42/

**Comment by petersomogyi** [comment]
Looks like hbase-spark-it can't be compiled on its own. IIRC we had this problem previously when the spark module was still in the main hbase repository.

File: spark/hbase-spark-it/pom.xml [hunk 1]
```diff
@@ -103,7 +103,8 @@
         <artifactId>maven-failsafe-plugin</artifactId>
         <configuration>
           <skip>false</skip>
-          <forkMode>always</forkMode>
+          <forkCount>1</forkCount>
+          <reuseForks>true</reuseForks>
```
Comment (petersomogyi) [inline spark/hbase-spark-it/pom.xml]: The [documentation](https://maven.apache.org/surefire/maven-failsafe-plugin/examples/fork-options-and-parallel-execution.html) at _Migrating the Deprecated forkMode Parameter to forkCount and reuseForks_ says forkMode=always is equivalent with forkCount=1 (default) and reuseForks=false. However, not reusing the processes would add an overhead so I think it is a reasonable modification.

Comment (dbist) [reply spark/hbase-spark-it/pom.xml]: @petersomogyi in terms of compiling hbase-spark-it individually, I ran mvn clean install on the module first and then I'm able to run mvn clean package on it, both with and without my patch.

**Comment by petersomogyi** [comment]
Yes, it this problem is not related to your change. I'm +1 on this and we can have a follow up issue for hbase-spark-it individual compile problem.

**Pull request merged**